tensorflow_addons/metrics/tests/geometric_mean_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests GeometricMean metrics."""
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from tensorflow_addons.metrics import GeometricMean
+
+
+def get_test_data():
+    return [
+        ([0, 0, 0, 0, 0, 0, 0, 0, 0, 0], 0),
+        ([0, 0, 0, 0, 0, 0, 0, 1, 2, 6], 0),
+        ([0.2, 0.5, 0.3, 0.6, 0.1, 0.7], 0.32864603),
+        ([8, 4, 1, 7, 2, 11, 9, 22, 52], 7.1804023),
+        ([8.2, 9.7, 9.1, 2.7, 1.1, 2.0], 4.0324492),
+        ([0.6666666, 0.215213, 0.15167], 0.27918512),
+    ]
+
+
+def assert_result(expected, result):
+    np.testing.assert_allclose(expected, result, atol=1e-6)
```

Comment:
this seems just a one line wrapper of `np.testing.assert_allclose`, we could probably use it directly.

Reply:
This is a wrapper who uses it to fix `atol`. Personally, I think this is better.

Reply:
ok, then do what you want as this doesn't matter at all 😃

Reply:
Thank You! 😄